src/pyhf/infer/mle.py

```diff
@@ -21,13 +21,53 @@ def fit(data, pdf, init_pars=None, par_bounds=None, **kwargs):
     """
     Run a unconstrained maximum likelihood fit.
 
+    Example:
```

Comment:
out of curiosity whe are we showing this example with minuit?

Reply:
I agree. for simplicity in docstrings, we could just drop the minuit portion, since it's also having issues.

Reply:
The reason here was to try to demo some of the functionality of the `kwargs` that exist for `fit`. Do you want this removed?

Reply:
`return_fitted_val=True` should work with any backend prob

Reply:
It does, right? But you're saying that as `return_uncertainties=True` doesn't work on all not to demo in the docs examples? That makes sense.